app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -0,0 +1,91 @@
+package com.github.se.cyrcle.model.parking
+
+import android.util.Log
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.firestore.FirebaseFirestore
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+
+/**
+ * ViewModel for the Parking feature.
+ *
+ * @param imageRepositoryCloudStorage the repository for the Parking feature
+ */
+class ParkingViewModel(
+    private val imageRepositoryCloudStorage: ImageRepository,
+    private val parkingRepositoryFirestore: ParkingRepository
```

Comment:
Attribute names should not be dependent on the implementation of the repository

Reply:
Fixed with new commit